README.md

```diff
@@ -13,13 +13,18 @@ To install:
 pip install --user scikit-hep-testdata
 ```
 
-Once installed, absolute file paths can be resolved using the build in helper method:
-```
+Once installed, absolute file paths can be resolved using the helper methods:
+``` python
 from skhep_testdata import data_path
 
 filename = data_path("some_file.root", raise_missing=True)
 ```
 
+or directly from the command-line:
```

Comment:
The example line
`filename = data_path("some_file.root", raise_missing=True)`
reminds me: `raise_missing` should be made default, no? After all, by default, one does want to make sure the requested file exists and is retrieved.

Reply:
Yes, you're right, and as of this PR that's actually the case now.  I left the argument visible in the README as a form of documenting it exists, but I could take it out here for simplicity and it would behave identically:
```
filename = data_path("some_file.root")
```

Reply:
Oops, I missed the change. Am browsing too quick it seems :S. All good, thanks.